app/src/lib/stores/github-user-store.ts

```diff
@@ -177,7 +177,7 @@ export class GitHubUserStore extends BaseStore {
   public async _loadAndCacheUser(
     accounts: ReadonlyArray<Account>,
     repository: Repository,
-    sha: string | null,
```

Comment:
We can drop the `| null` here because there's no longer any caller that is passing in `null`, so we can simplify the code paths in `findUserWithAPI`.